pages/dr8/description.rst

```diff
@@ -154,27 +154,35 @@ and raw data can be obtained through `the NOAO portal`_ (or via ftp; see also th
 Sections of the Legacy Surveys  for DR8 can be obtained as JPEGs or FITS files using
 the cutout service, for example, as follows:
 
-JPEG (`DECaLS`_): http://legacysurvey.org/viewer/jpeg-cutout?ra=190.1086&dec=1.2005&layer=decals-dr8&pixscale=0.27&bands=grz
+JPEG: http://legacysurvey.org/viewer/jpeg-cutout?ra=190.1086&dec=1.2005&layer=dr8&pixscale=0.27&bands=grz
 
-FITS (`DECaLS`_): http://legacysurvey.org/viewer/fits-cutout?ra=190.1086&dec=1.2005&layer=decals-dr8&pixscale=0.27&bands=grz
+FITS: http://legacysurvey.org/viewer/fits-cutout?ra=190.1086&dec=1.2005&layer=dr8&pixscale=0.27&bands=grz
 
-JPEG (`BASS`_/`MzLS`_): http://legacysurvey.org/viewer/jpeg-cutout?ra=154.7709&dec=46.4537&layer=mzls+bass-dr8&pixscale=0.27&bands=grz
+This will merge the northern (MzLS+BASS) and equatorial (DECam) images at the Dec=32.375 line.
 
-FITS (`BASS`_/`MzLS`_): http://legacysurvey.org/viewer/fits-cutout?ra=154.7709&dec=46.4537&layer=mzls+bass-dr8&pixscale=0.27&bands=grz
+To request images from only the northern or equatorial surveys, specify :math:`dr8-north` or :math:`dr8-south`, for example:
```

Comment:
Suggestion:
```
specify `dr8-north` or `dr8-south`
```
instead of: 
```
specify :math:`dr8-north` or :math:`dr8-south`
```
So the dashes aren't rendered as minus signs.

Reply:
done in 545a489